src/pudl_archiver/archivers/epacems.py

```diff
@@ -57,6 +57,7 @@ class EpaCemsArchiver(AbstractDatasetArchiver):
 
     name = "epacems"
     concurrency_limit = 2  # Number of files to concurrently download
+    allowed_file_rel_diff = 0.35  # Set higher tolerance than standard
```

Comment:
Is there a particular reason why we want to set this tolerance higher? bc the files are big and few?

Reply:
Yes, if we have one quarter of data and we add a second quarter it'll be substantially more than a 25% increase in file size